tests/test_persistence.py

```diff
@@ -1638,32 +1642,6 @@ def test_recalc_trade_from_orders_ignores_bad_orders(fee):
     assert trade.open_trade_value == 2 * o1_trade_val
     assert trade.nr_of_successful_buys == 2
 
-    # Just to make sure sell orders are ignored, let's calculate one more time.
-    sell1 = Order(
```

Comment:
I'd want to have this kept there - as it can test that the entry price will not be modified by exits.

It's possible that some asserts below no longer make sense - but the case itself is valid and should be kept.

On the contrary to removing this - i'd want to have an additional test showing more complex behaviours (like mixing/matching of buys/sells - specifically testing the behavior of the calculation part.
While the test in `test_freqtradebot` is good - it also includes the whole "bot complexity" - whereas here we can focus purely on the calculation.

Reply:
removed as it is tested already in previous test. Added again.